pkg/controlplane/instance.go

```diff
@@ -259,7 +283,11 @@ func (cp *Instance) DeleteExport(name string) (*cpstore.Export, error) {
 		}
 	}
 
-	if err := cp.xdsManager.DeleteExport(name); err != nil {
+	namespacedName := types.NamespacedName{
+		Name:      name,
+		Namespace: cp.namespace,
+	}
+	if err := cp.xdsManager.DeleteExport(namespacedName); err != nil {
```

Comment:
Please be consistent here and in L379 (i.e., create and pass NamespacedName in param list or not...)